pkg/ingester/lifecycle_test.go

```diff
@@ -94,11 +95,23 @@ func TestIngesterTransfer(t *testing.T) {
 	limits, err := validation.NewOverrides(defaultLimitsTestConfig())
 	require.NoError(t, err)
 
+	tokenDir1, err := ioutil.TempDir(os.TempDir(), "ingester_transfer")
```

Comment:
Could you explain why we need to add `TokensFilePath` config option to these tests? Looks it make tests harder to read without adding any value to the tests itself.

Reply:
1. To test that everything works with tokens on disk (I think we need tests for both on and off disk).
2. So that we can clean it up later. Else it leaves behind the token files in the project (like the ones I accidentally pushed)

Reply:
Yeah, I realized now that we don't need it. Earlier I thought that all the functionalities need to be tested with tokens on disk to make sure that it works fine. But figured that testing only the tokens on disk would suffice.